test/gpu/pack_int8_args.cpp

```diff
@@ -81,7 +69,7 @@ TEST_CASE(quant_dot)
         return m;
     };
 
-    auto create_optimized_int8_x4 = [](bool int8_x4) {
+    auto create_optimized_int8_x4 = [](bool int8_x4, bool compute_fp32) {
```

Comment:
Does it make sense to have this a parameter to the lambda? Wouldn't it be better to just to remove the parameter and refactor the function in L95 to something like this?
```
{{"int8_x4_format", int8_x4}, {"compute_fp32", migraphx::gpu::get_compute_fp32_flag()}}),
```

Reply:
yes, 
Made changes.